docs/develop/xml-manifest-overview.md

```diff
@@ -287,84 +287,87 @@ The following sections show examples of manifest v1.1 XML files for content, tas
-  <!-- The display name of your add-in. Used on the store and various placed of the Office UI such as the add-ins dialog -->
+  <!-- The display name of your add-in. Used on the store and various placed of the Office UI such as the add-ins dialog. -->
   <DisplayName DefaultValue="Add-in Commands Sample" />
   <Description DefaultValue="Sample that illustrates add-in commands basic control types and actions" />
-  <!--Icon for your add-in. Used on installation screens and the add-ins dialog -->
+  <!-- Icon for your add-in. Used on installation screens and the add-ins dialog. -->
   <IconUrl DefaultValue="https://contoso.com/assets/icon-32.png" />
   <HighResolutionIconUrl DefaultValue="https://contoso.com/assets/hi-res-icon.png" />
   <SupportUrl DefaultValue="[Insert the URL of a page that provides support information for the app]" />
-  <!--BeginTaskpaneMode integration. Any client that doesn't understand commands will use this section.
-    This section will also be used if there are no VersionOverrides -->
+  <!-- End Basic Settings. -->
+
+  <!-- BeginTaskpaneMode integration. Any client that doesn't understand commands will use this section.
+    This section will also be used if there are no VersionOverrides. -->
   <Hosts>
     <Host Name="Document"/>
   </Hosts>
   <DefaultSettings>
     <SourceLocation DefaultValue="https://commandsimple.azurewebsites.net/Taskpane.html" />
   </DefaultSettings>
-  <!--EndTaskpaneMode integration -->
+  <!-- EndTaskpaneMode integration. -->
 
   <Permissions>ReadWriteDocument</Permissions>
 
-  <!--BeginAddinCommandsMode integration-->
+  <!-- BeginAddinCommandsMode integration. -->
   <VersionOverrides xmlns="http://schemas.microsoft.com/office/taskpaneappversionoverrides" xsi:type="VersionOverridesV1_0">
     <Hosts>
-      <!--Each host can have a different set of commands. Cool huh!? -->
-      <!-- Workbook=Excel Document=Word Presentation=PowerPoint -->
-      <!-- Make sure the hosts you override match the hosts declared in the top section of the manifest -->
+      <!-- Each host can have a different set of commands. Cool huh!? -->
+      <!-- Workbook=Excel, Document=Word, Presentation=PowerPoint -->
+      <!-- Make sure the hosts you override match the hosts declared in the top section of the manifest. -->
       <Host xsi:type="Document">
-        <!-- Form factor. Currently only DesktopFormFactor is supported. We will add TabletFormFactor and PhoneFormFactor in the future-->
+        <!-- Form factor. DesktopFormFactor is supported. Other form factors are available depending on the host and feature. -->
         <DesktopFormFactor>
-          <!--Function file is an html page that includes the javascript where functions for ExecuteAction will be called.
-            Think of the FunctionFile as the "code behind" ExecuteFunction-->
+          <!-- Function file is an HTML page that includes the JavaScript where functions for ExecuteAction will be called.
```

Comment:
The HTML file can have a `<script>` tag that loads the JavaScript, so consider this change:

```suggestion
          <!-- Function file is an HTML page that includes, or loads, the JavaScript where functions for ExecuteAction will be called.
```